sympy/integrals/risch.py

```diff
@@ -680,6 +691,14 @@ def decrement_level(self):
         return None
 
 
+def update_sets(seq, atoms, func):
+    s = set(seq)
+    s = atoms.intersection(s)
```

Comment:
I just swapped this line and the one below this, though result-wise it wouldn't matter.